.github/workflows/publish-rc.yml

```diff
@@ -48,33 +48,13 @@ jobs:
     needs: tag
     if: needs.tag.outputs.is_rc == 'true'
     steps:
-      # <common-build> - Uses YAML anchors in the future
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0 # Needs full depth for changelog generation
-      - uses: actions/setup-node@v4
-        with:
-          node-version: 20
-          check-latest: true
```

Comment:
Only thing missing would be this check. Could you pass this down to the action as well.

Reply:
Sorry I didn't remember if `build-setup` action already have `check-latest`. Please ignore this comment.